app/controllers/course-admin/code-examples-index.ts

```
@@ -0,0 +1,61 @@
+
+export default class CodeExamplesIndexController extends Controller {
+  @service declare router: RouterService;
+  
+  queryParams = ['language'];
+  language: string | null = null;
+  
+  declare model: ModelType;
+  
+  get stagesWithAnalyses(): StageWithAnalysis[] {
+    const stagesWithAnalyses: StageWithAnalysis[] = [];
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)